spec/ddtrace/contrib/rspec/instrumentation_spec.rb

```diff
@@ -5,36 +5,35 @@
 require 'ddtrace'
 
 RSpec.describe 'RSpec hooks' do
-  extend ConfigurationHelpers
-
   let(:configuration_options) { {} }
 
-  around do |example|
-    old_configuration = ::RSpec.configuration
-    ::RSpec.configuration = ::RSpec::Core::Configuration.new
-
+  before do
     Datadog.configure do |c|
       c.use :rspec, configuration_options
     end
-    example.run
+  end
+
+  # Yields to a block in a new RSpec global context. All RSpec
+  # test configuration and execution should be wrapped in this method.
+  def with_new_rspec_environment
+    old_configuration = ::RSpec.configuration
+    old_world = ::RSpec.world
+    ::RSpec.configuration = ::RSpec::Core::Configuration.new
+    ::RSpec.world = ::RSpec::Core::World.new
 
-    RSpec.configuration = old_configuration
-    Datadog.configuration.reset!
+    yield
+  ensure
+    ::RSpec.configuration = old_configuration
+    ::RSpec.world = old_world
   end
 
   it 'creates span for example' do
-    spans = []
-    spec = RSpec.describe 'some test' do
-      let(:active_span) { Datadog.configuration[:rspec][:tracer].active_span }
-
-      it 'foo' do
-        spans << active_span
-      end
+    spec = with_new_rspec_environment do
+      RSpec.describe 'some test' do
+        it 'foo' do; end
+      end.tap(&:run)
     end
-    spec.run
 
-    expect(spans.count).to eq(1)
-    span = spans.first
     expect(span.span_type).to eq(Datadog::Ext::AppTypes::TEST)
```

Comment:
These implicit helpers get me confused every time, because they show up from nowhere -- I need to grep the codebase to understand where they coming from.

It's a bit minor, but I'd really like if we could call them explicitly `TracerHelpers.span` (it's really obvious where it's coming from) or we had an explicit `include TracerHelpers` in this file.

Reply:
`TracerHelpers.span` everything is too verbose, `include TracerHelpers` could be nice.